### ISSUE TITLE
App version file path should include app name

### ISSUE THREAD

**Issue by miyconst** [Starcounter/Warehouse#18]
Currently applications are saved in the next path:

```
c:\warehouse\storage\{OrganizationName}\versions\SignIn.zip
```

This might be an issue if an organization has lots of applications.

I propose to change the path to:

```
c:\warehouse\storage\{OrganizationName}\versions\{AppName}\SignIn-{AppVersion}.zip
```

**Comment by andwah** [comment]
True,, and i will also add a folder in the path for the `channel`

`c:\warehouse\storage\{OrganizationName}\versions\{AppName}\{Channel}\SignIn-{AppVersion}.zip`

**Comment by andwah** [comment]
Fixed 2933c4cb03770f498f1e80904710ebd93e4f0fc8
